tests/test_verify_registration_response.py

```diff
@@ -87,9 +88,7 @@ def test_raises_exception_on_unsupported_attestation_type(self) -> None:
         rp_id = "localhost"
         expected_origin = "http://localhost:5000"
 
-        with self.assertRaisesRegex(
-            Exception, "value is not a valid enumeration member"
-        ):
+        with self.assertRaises(ValidationError):
```

Comment:
The message is different in v1 and v2 so I just changed it to assert a validation error is raised